qiskit/opflow/evolutions/trotterizations/suzuki.py

```diff
@@ -60,6 +60,8 @@ def convert(self, operator: OperatorBase) -> OperatorBase:
 
         if isinstance(operator.coeff, (float, ParameterExpression)):
             coeff = operator.coeff
+            if not isinstance(coeff, float):
```

Comment:
A positive condition is clearer than a negative condition:
```python
if isinstance(coeff, ParameterExpression):
```

Reply:
I got a DeprecationWarning: The class ``qiskit.opflow.evolutions.trotterizations.suzuki.Suzuki`` is deprecated as of qiskit-terra 0.24.0. According to the code migration guidelines, visit https://qisk.it/opflow_migration class qiskit.synthesis.SuzukiTrotter could be used. So I propose to this class instead of the opflow module.